### PR TITLE
adds parameter to setup VideobridgeExpireThread sleep time

### PR DESCRIPTION
**Pull request by sidorovis**

### CODE REVIEW

File: src/main/java/org/jitsi/videobridge/VideobridgeExpireThread.java [hunk 1]
```diff
@@ -44,6 +47,23 @@
      */
     private final WeakReference<Videobridge> videobridge;
 
+    /**
+     * Property name for EXPIRE_CHECK_SLEEP_SEC for
+     * {@link #expireCheckSleepSec}.
+     */
+    private static final String EXPIRE_CHECK_SLEEP_SEC
```
Comment (bgrozev) [inline src/main/java/org/jitsi/videobridge/VideobridgeExpireThread.java]: This should be public (convention for all property names)

File: src/main/java/org/jitsi/videobridge/VideobridgeExpireThread.java [hunk 2]
```diff
@@ -18,7 +18,10 @@
 import java.lang.ref.*;
 import java.util.*;
 
+import org.jitsi.osgi.ServiceUtils2;
```
Comment (bgrozev) [inline src/main/java/org/jitsi/videobridge/VideobridgeExpireThread.java]: Please use * imports

**Comment by bgrozev** [comment]
LGTM, except for the 2 minor comments I left

**Comment by sidorovis** [comment]
Updated according to comments.

File: src/main/java/org/jitsi/videobridge/VideobridgeExpireThread.java [hunk 3]
```diff
@@ -44,6 +47,23 @@
...
+     * Property name for EXPIRE_CHECK_SLEEP_SEC for
+     * {@link #expireCheckSleepSec}.
+     */
+    public static final String EXPIRE_CHECK_SLEEP_SEC
+            = "org.jitsi.videobridge.EXPIRE_CHECK_SLEEP_SEC";
+
+    /**
+     * Default value for {@link #expireCheckSleepSec}.
+     */
+    private static final int EXPIRE_CHECK_SLEEP_SEC_DEFAULT = 60;
```
Comment (gpolitis) [inline src/main/java/org/jitsi/videobridge/VideobridgeExpireThread.java]: Not really mandatory, but maybe we could replace the 60 with Channel.DEFAULT_EXPIRE.

File: src/main/java/org/jitsi/videobridge/Videobridge.java [hunk 4]
```diff
@@ -1335,6 +1341,9 @@ void start(final BundleContext bundleContext)
                     bundleContext,
                     ConfigurationService.class);
 
+        videobridgeExpireThread.start(bundleContext);
+        videobridgeExpireThread.start();
```
Comment (gpolitis) [inline src/main/java/org/jitsi/videobridge/Videobridge.java]: Not really mandatory, but what do you guys think if we called `start` from within `start(bundleContext)`?

**Comment by bgrozev** [comment]
jenkins please test this

**Comment by bgrozev** [comment]
jenkins test this please (thanks, Damencho...)

**Pull request merged**